### ISSUE TITLE
Request the release of your pretrain model   

### ISSUE THREAD

**Issue by YCaigogogo** [FactoDeepLearning/MultitaskVLFM#1]
Dear developers,
  Hello, I would like to ask if you can open the download path of the trained model used in your paper, thank you very much!

**Comment by FactoDeepLearning** [comment]
Here it is: https://zenodo.org/record/8124014

**Comment by YCaigogogo** [comment]
May I ask whether the text encoder of clip in this pre-trained model has been fine-tuned by the method in the paper?

**Comment by FactoDeepLearning** [comment]
Yes it has been fine-tuned (not all layers but only the last transformer layer + the projection layers)

**Comment by YCaigogogo** [comment]
Thank you for your reply. And I've got what might be a simple problem. I found this error message when I try to run "python3 Multitasking/main.py --load-pretrain=True --train=False". And My current path is "/data/yic/MultitaskVLFM-main". It is strange. How can I solve it. 

=================================================================
Traceback (most recent call last):
  File "/data/yic/MultitaskVLFM-main/Multitasking/main.py", line 4, in <module>
    from Multitasking.dataset_utils.cub200_dataset import CUBDataset, check_or_download_CUB200_dataset, check_or_download_CUB200_oracle_seg
ModuleNotFoundError: No module named 'Multitasking'

**Comment by FactoDeepLearning** [comment]
As stated in the readme, "pip install -e ." Should solve this issue
